docs/development.md

```diff
@@ -2,37 +2,37 @@
 
 ## Running with CRDs only on kubernetes cluster
 
-The minimial compoenent which must be deployed on your local kubernetes cluster is testkube-operator with project CRDs (<https://github.com/kubeshop/testkube-operator>)
+The minimial component which must be deployed on your local kubernetes cluster is testkube-operator with project CRDs (<https://github.com/kubeshop/testkube-operator>)
 
-Checkout this project and run:
+Checkout testkube-operator project and run:
 
 ```sh
 make install 
 ```
 
-to install CRD's in your local cluster
+to install CRD's into your local cluster
 
 ## Running on local machine
 
-Next critical component is API (<https://github.com/kubeshop/testkube>) and some executor you can build your
-own tests executor or use one from TestKube.
+Next critical component is testkube API (<https://github.com/kubeshop/testkube>) and some executor you can build - your
+own tests executor or existing one from TestKube.
 
-First let's run local API server:
+Checkout testkube project and run local API server:
```

Comment:
You already checked it out on line 7

Reply:
On line 7, we checkout testkube-operator, so guess, it's worth to clearly state that we checkout different projects